src/launch-apps-from-collect.rst

```diff
@@ -107,24 +144,10 @@ XLSForm
-
-.. _launch-apps-uri-data:
-
-Specifying a URI as intent data
----------------------------------
-
-Since Collect v1.16.0, the value for the reserved parameter name ``uri_data`` is converted to a URI and used as the data for the intent. The intent data determines which application to launch when using implicit intents such as `SENDTO <https://developer.android.com/reference/android/content/Intent#ACTION_SENDTO>`_. For example:
-
-``ex:android.intent.action.SENDTO(uri_data='smsto:5555555', sms_body=${message})``
-  Launches a new message in an SMS app with the destination number set to ``5555555`` and the message body set to the contents of the ``message`` field.
+The ``intent`` attribute is only used when the group has an ``appearance`` of ``field-list``. The format and the functionality of the ``intent`` value is the same as above. If the ``Intent`` extras returned by the external application contains values with keys that match the type and the name of the sub-fields, then the values from the ``Intent`` extras overwrite the current values of those sub-fields.
 
-``ex:android.intent.action.SENDTO(uri_data='mailto:example@example.com?subject=${subject}&body=${message})``
-  Launches a new message in an email app with destination address set to ``example@example.com``, the subject set to the contents of the ``subject`` field and the body set to the contents of the ``message`` field.
+The external app is launched with the parameters that are defined in the intent string plus the values of all the sub-fields that are either text, decimal, integer, or binary (filename is sent). Any other sub-field is invisible to the external app.
 
-``ex:android.intent.action.DIAL(uri_data='tel:5555555')``
-  Launches a phone dialer with the number ``5555555`` as the number to dial.
+Since Collect v1.30.0, it is possible to populate binary fields (:ref:`image <external-app-image-widget>`, :ref:`video <external-app-video-widget>`, :ref:`audio <external-app-audio-widget>` or :ref:`file <external-app-file-widget>`) in field lists. An app that returns one or more binary files to Collect as part of a field list must provide a `content URI <https://developer.android.com/guide/topics/providers/content-provider-basics#ContentURIs>`_ as the value for each ``extra`` that corresponds to a binary question to populate. Additionally, the external application must specify `ClipData items <https://developer.android.com/reference/android/content/ClipData.Item>`_ for each URI it returns. Your app can then grant Collect temporary permissions to the files using the `Intent.FLAG_GRANT_READ_URI_PERMISSION <https://developer.android.com/reference/android/content/Intent#FLAG_GRANT_READ_URI_PERMISSION>`_ intent flag. See :ref:`a similar code example above <external-app-design>`.
```

Comment:
Links here are to external widgets but in such an intent group we can use normal binary widgets and also normal binary widgets are used as a sample in the table above so it might be a bit misleading.